include/module.modulemap

```diff
@@ -0,0 +1,66 @@
+module rcheevos {
+  umbrella header "rcheevos.h"
+
+  module rc_api_editor {
+	  header "rc_api_editor.h"
+	  export *
+  }
+  
+  module rc_api_info {
+	  header "rc_api_info.h"
+	  export *
+  }
+
+  module rc_api_request {
+	  header "rc_api_request.h"
+	  export *
+  }
+  
+  module rc_api_runtime {
+	  header "rc_api_runtime.h"
+	  export *
+  }
+
+  module rc_api_user {
+	  header "rc_api_user.h"
+	  export *
+  }
```

Comment:
I'm not really familiar with how this works, but from a quick Google, it seem like you'd want to group the "rc_api_" files under a single "rapi" module, and then everything else except "rc_hash" and "rc_url" in a "runtime" module.